src/FormMigration/Steps/FormFieldManager.php

```diff
@@ -0,0 +1,352 @@
+            ]
+        ]);
+    }
+
+    private function addEmailField($field): BlockModel
+    {
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/email',
+        ]);
+    }
+
+    private function addFileUploadField($field): BlockModel
+    {
+        $allowedFileTypes = array_map(function ($type) {
+            switch ($type) {
+                case 'images': return 'image';
+                default: return $type;
+            }
+        }, $field['extension']);
+
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/file-upload',
+            'attributes' => [
+                'maxFileSize' => $field['max_size'],
+                'allowedFileTypes' => $allowedFileTypes,
+            ]
+        ]);
+    }
+
+    private function addHiddenField($field): BlockModel
+    {
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/hidden',
+        ]);
+    }
+
+    private function addHtmlField($field): BlockModel
+    {
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/html',
+            'attributes' => [
+                'htmlCode' => $field['html'],
+            ]
+        ]);
+    }
+
+    private function addMultiSelectField($field): BlockModel
+    {
+        $fieldType = $field['template'] === 'checkbox_field' ? 'checkbox' : 'dropdown';
+        $options = array_map(function ($option) use ($field) {
+            return [
+                'label' => $option,
+                'value' => '',
+                'checked' => in_array($option, $field['selected'], true),
+            ];
+        }, array_filter($field['options']));
+
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/multi-select',
+            'attributes' => [
+                'fieldType' => $fieldType,
+                'options' => $options,
+            ]
+        ]);
+    }
+
+    private function addPhoneField($field): BlockModel
+    {
+        $phoneFormat = $field['format'] === 'domestic' ? 'domestic' : 'unformatted';
+
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/phone',
+            'attributes' => [
+                'phoneFormat' => $phoneFormat,
+            ]
+        ]);
+    }
+
+    private function addRadioField($field): BlockModel
+    {
+        $options = array_map(function ($option) use ($field) {
+            return [
+                'label' => $option,
+                'value' => '',
+                'checked' => $option === $field['selected'],
+            ];
+        }, array_filter($field['options']));
+
+        return BlockModel::make([
+            'name' => 'givewp-form-field-manager/radio',
+            'attributes' => [
+                'options' => $options,
+            ]
+        ]);
+    }
+
+    private function addTextField($field): BlockModel
+    {
+        return BlockModel::make([
+            'name' => 'givewp/text',
```

Comment:
@kjohnson I cannot reproduce this issue. It may have been addressed on df57dd557b2b329ce458530aebc8b0a8e52ec9a0 and merged to this branch when I updated it with develop.

Reply:
Ok. Looks good now.